src/cmd/cli/command/deploymentinfo.go

```diff
@@ -44,12 +45,28 @@ func printServiceStatesAndEndpoints(serviceInfos []*defangv1.ServiceInfo) error
 				showCertGenerateHint = true
 			}
 		}
+		endpoints := make([]string, 0, len(serviceInfo.Endpoints))
+		for _, endpoint := range serviceInfo.Endpoints {
+			if endpoint == "" {
+				continue
+			}
+
+			if !regexp.MustCompile(`\.internal:\d{1,5}$`).MatchString(endpoint) {
```

Comment:
This feels brittle. There's some code elsewhere that mirrors this. Perhaps we just check for the port number: no port = https (which is true)

Reply:
will fail on GCP PG